README.md

```diff
@@ -1 +1,33 @@
-# aqts-capture-ts-corrected
+# Aquarius Timeseries (AQTS) Pre-Process Timeseries Data
+
+[![Build Status](https://travis-ci.com/usgs/aqts-capture-ts-corrected.svg?branch=master)](https://travis-ci.com/usgs/aqts-capture-ts-corrected)
+[![codecov](https://codecov.io/gh/usgs/capture-ts-corrected/branch/master/graph/badge.svg)](https://codecov.io/gh/usgs/aqts-capture-ts-corrected)
```

Comment:
In the first codecov url on line 4, sub in `aqts-capture-ts-corrected` in place of `capture-ts-corrected`